core/templates/services/context.service.spec.ts

```diff
@@ -213,6 +282,12 @@ describe('Context service', () => {
       () => {
         expect(ecs.canAddOrEditComponents()).toBe(true);
       });
+
+    it('should correctly affirm that page contains reference to skills',
```

Comment:
Duplicate description. Please make this more specific.

Reply:
Done!